src/common.speech/SynthesisAdapterBase.ts

```diff
@@ -146,9 +144,11 @@ export class SynthesisAdapterBase implements IDisposable {
         this.connectionEvents.attach((connectionEvent: ConnectionEvent): void => {
             if (connectionEvent.name === "ConnectionClosedEvent") {
                 const connectionClosedEvent = connectionEvent as ConnectionClosedEvent;
-                this.cancelSynthesisLocal(CancellationReason.Error,
-                    connectionClosedEvent.statusCode === 1007 ? CancellationErrorCode.BadRequestParameters : CancellationErrorCode.ConnectionFailure,
-                    connectionClosedEvent.reason + " websocket error code: " + connectionClosedEvent.statusCode);
+                if (connectionClosedEvent.statusCode !== 1000) {
```

Comment:
What happens in this flow if the statusCode 1000 is received? We silently ignore the closed event?

Reply:
Thanks for reviewing. `statusCode` 1000 means normal closure, which is normally caused by client close the connection. The error code info could be found in our [internal wiki ](https://speechwiki.azurewebsites.net/partners/protocol-websockets-tts.html?q=websocket)